sui/types-utils.js

```diff
@@ -21,6 +22,11 @@ const bytes32Struct = bcs.fixedArray(32, bcs.u8()).transform({
     output: (id) => hexlify(id),
 });
 
+const UID = bcs.fixedArray(32, bcs.u8()).transform({
```

Comment:
also add types to cgp-sui repo since we're migrating to it

Reply:
https://github.com/axelarnetwork/axelar-cgp-sui/pull/74